app_widgetbook/lib/core/tweet/collapsed_tweet.dart

```diff
@@ -1,7 +1,7 @@
-import 'package:app_widgetbook/dummy_data/dummy_media.dart';
+import 'package:app_widgetbook/dummy_data/dummy_tweets.dart';
 import 'package:app_widgetbook/dummy_data/dummy_users.dart';
+import 'package:app_widgetbook/utils.dart';
 import 'package:core/core.dart';
-import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
```

Comment:
<img width="406" alt="image" src="https://user-images.githubusercontent.com/16862002/201635490-c99151db-cb03-4cef-84b2-494320b41101.png">
do you also see the rectangles?

Reply:
This is happening in the Cloud not sure if it also appears locally.

Reply:
Oh yes I noticed that. This seems to only happen on web. I will look into it in the next PR